CMake/gkd350h_defs.cmake

```diff
@@ -7,6 +7,8 @@ set(SDL1_VIDEO_MODE_HEIGHT 240)
 
 set(SDL1_VIDEO_MODE_BPP 16)
 set(PREFILL_PLAYER_NAME ON)
+set(MULTIPLAYER_DEFAULT_IP "\"10.1.1.1\"")
+set(MULTIPLAYER_DEFAULT_PASSWORD "\"123\"")

```

Comment:
Setting default IP and password at build time doesn't seam appropriate.

Reply:
There isn't another place to set it. The packages for these devices don't have an install step, they create an ini file at startup if one doesn't exist.

We *could* generate a random one at startup as an alternative.

Reply:
Imo an onscreen keyboard is the right solution, that would also solve the issue of a missing input when splitting gold.

Reply:
Here is what is being used for the Vita port: https://github.com/diasurgical/devilutionX/tree/4c2a43b23e92adcb1d047fdd52e10fb53c9471ff/vita/danzeffSDL maybe we can integrate that.

![billede](https://user-images.githubusercontent.com/204594/74593015-7aa80600-5027-11ea-81d8-9b16da9c76c5.png)

Reply:
OK, I have no interesting in implementing that so will leave this to someone else.

There is an SDL1 virtual keyboard here https://github.com/glebm/rs97-commander/blob/upd2/keyboard.cpp